### PR TITLE
Central app

### PR DESCRIPTION
**Pull request by stancl**

### CODE REVIEW

File: docs/source/v2/central-app.blade.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+## Central domains {#central-domains}
+
+However, since you don't want routes related to the app on your main domain and sign up forms on tenant domains, you must also define what domains host the central stuff in the `tenancy.exempt_domains` config.
+
+## Using central things inside the tenant app {#using-central-things-inside-the-tenant-app}
+
+To use central things (databases/caches/Redis connections/filesystems/...) on special places of your tenant app, you may do the following.
+
+## Central database {#central-database}
```
Comment (drbyte) [inline docs/source/v2/central-app.blade.md]: I wonder if this and the headings following it should use `###` and be treated as sub-headings of the one above?

**Comment by drbyte** [comment]
There's a section about `Using central things inside the tenant app`. Should the reverse also be in this document? ie: `using tenant things inside central`?

**Comment by stancl** [comment]
Ha, good point. I'll make that part of #26 after I release 2.1.0.

File: docs/source/v2/central-app.blade.md (hunk 1 above)
Comment (stancl) [reply docs/source/v2/central-app.blade.md]: Good catch.

**Pull request merged**